GitCommands/AsyncLoader.cs

```diff
@@ -94,7 +95,8 @@ public Task Load(Action<CancellationToken> loadContent, Action onLoaded)
                         loadContent(token);
                     }
                 }, token)
-                .ContinueWith((task) =>
+                .ContinueWith(
+                    (task) =>
```

Comment:
Can't say I like this change...
What rule is responsible for this?

Reply:
I believe it's [SA1116](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1116.md).

It's in the 'readability' section. My interpretation of the rationale here is that parameters should be aligned so that you don't accidentally miss the first one. I generally use this approach in my own code in those cases where the list is split across multiple lines, so maybe I just expect it.

In the example linked above I definitely feel like there's more chance of me mistaking the number of parameters in the first example.

It seems that there's no equivalent rule for argument lists though.

Reply:
I don't mind disabling it. The positive side of enabling it would be having a consistency across the code base enforced by the rule. Am I right thinking that without that rule it will be possible to write both: in the same line and in the next line?